app/services/instrumentation.rb

```diff
@@ -0,0 +1,34 @@
+module Instrumentation
+  class << self
+    def append_to_log(args = {})
+      RequestStore.store[:custom_log_items] ||= {}
+      RequestStore.store[:custom_log_items].merge!(args) unless args.blank?
+      RequestStore.store[:custom_log_items]
+    end
+    alias_method :custom_log_items, :append_to_log
+
+    def log(message, category = nil)
```

Comment:
Since this method has been moved into a class which is responsible for logging as well as timing, it's no longer clear that `.log` records timing information (`append_to_log` does not). I wonder if there's a clearer API for both of these methods now that they're in the same place?